sessions/session-b.md

```diff
@@ -0,0 +1,36 @@
+# Session B: Creating low-power sensor networks using LoRa
```

Comment:
Is there a shorter title you could have, with a longer description after a colon (for linking on the website more easily?
e.g. _Creating sensor networks: Using LoRA to deploy low-power/low-bandwidth wide area networks_  (totally just an example, but something in that format?